### DISCUSSION TITLE
I didn't encounter any errors on my local build, but when I deployed it on Vercel, I received an error message.

### DISCUSSION THREAD

**Discussion by MSR-07** [vercel/next.js#61878] [Help] [answered]
### Summary

I am currently using Next.js version 14.4 along with Next PWA.

![Screenshot 2024-02-10 032418](https://github.com/vercel/next.js/assets/102695412/62e57961-2ef0-49e0-9744-ce6cd474fb92)
![Screenshot 2024-02-10 032443](https://github.com/vercel/next.js/assets/102695412/3f929b0e-2cf6-42c6-91ee-876a500f0d34)

**Comment by DevDaniloFerrari** [comment, marked as answer]
What version of node are you using?
Try using version 20.x.

Also check that the casing of your file import matches what's on disk. Windows isn't case sensitive like our Linux build environments are!

**Reply by MSR-07** [reply]
my node version 20.10.0.

**Reply by MSR-07** [reply]
I've successfully resolved the issue – turns out it was related to a Git stash problem. Thank you for your suggestions; they were really helpful in guiding me towards the solution. Appreciate your assistance!☺

**Reply by DevDaniloFerrari** [reply]
Great!
